tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -152,4 +152,4 @@ def set_packet_props_full_relativity(r_packet, numba_model):
 
     r_packet.nu *= inverse_doppler_factor
     r_packet.energy *= inverse_doppler_factor
-    r_packet.mu = (r_packet.mu + beta) / (1 + beta * r_packet.mu)
+    r_packet.mu = (r_packet.mu + beta) / (1 + beta * r_packet.mu)
```

Comment:
Remove this change

Reply:
I am unsure how this got changed, sorry. Same as the one above, I don't remember changing it.

Reply:
I am unsure why this was highlighted, as when I go to copy the old code it won't commit as it's the same exact code. However, I added a newline at the end of each file, as that is what Black does for format and it what TARDIS uses.